### ISSUE TITLE
Electron Compile does not compile files with extension .JSX

### ISSUE THREAD

**Issue by mfunkie** [electronjs/electron-compile#15]
In the Yeoman app, after setting up everything with Babel Compile.  I noticed that I was getting an `Unexpected reserved word` for the token `import` in the statement `import React from 'react'` in my app.jsx file required in through a main.js file.  By renaming the extension to `.js`, the issue went away, but then other files started to fail because they have the `.jsx` extension.

As far as I know there's no extension option within Babel to enforce this that I can provide, and `.jsx` is a standard extension in the React community.

**Comment by paulcbetts** [comment]
@mfunkie If you can get me a valid an an invalid JSX file (i.e. a JSX file that throws _any_ error), I can probably patch this in really easily

**Comment by mfunkie** [comment]
Do you mean like this? Or do you want physical files?

Valid:

```
import React from 'react';

export default React.createClass({
  render() {
    return <div>Hello World!</div>
  }
});
```

Invalid (unclosed div JSX tag)

```
import React from 'react';

export default React.createClass({
  render() {
    return <div>Hello World!<div>
  }
});
```

If you just mean any .jsx file that throws an error with electron-compile, the first one would fail too.

**Comment by paulcbetts** [comment]
@mfunkie Perfect, I'll see if I can add that as a test case

**Comment by paulcbetts** [comment]
Fixed in #16!

**Comment by mfunkie** [comment]
That was fast. Thank you!
